packages/mongodb-chatbot-server/src/routes/responses/responsesRouter.test.ts

```diff
@@ -0,0 +1,41 @@
+import request from "supertest";
+import { AppConfig } from "../../app";
+import { DEFAULT_API_PREFIX } from "../../app";
+import { makeTestApp } from "../../test/testHelpers";
+import { makeTestAppConfig } from "../../test/testHelpers";
+
+jest.setTimeout(60000);
+
+describe("Responses Router", () => {
+  const ipAddress = "127.0.0.1";
+  const responsesEndpoint = DEFAULT_API_PREFIX + "/responses";
+  const validRequestBody = {
+    model: "mongodb-chat-latest",
```

Comment:
same nit as the other test about not using string literal here, instead pass from config